src/adam_impact_study/impacts_study.py

```diff
@@ -108,9 +156,26 @@ def run_impact_study_fo(
         )
     physical_params_df = pd.DataFrame(data)
 
+    obj_id = impactor_orbit.object_id[0]
+
+    print("Object ID: ", obj_id)
+
+    sorcha_config_file_name = f"sorcha_config_{RUN_NAME}_{obj_id}.ini"
+    sorcha_orbits_file = f"sorcha_input_{RUN_NAME}_{obj_id}.csv"
+    sorcha_physical_params_file = f"sorcha_params_{RUN_NAME}_{obj_id}.csv"
+    sorcha_output_name = f"sorcha_output_{RUN_NAME}_{obj_id}"
+    sorcha_output_file = f"{sorcha_output_name}.csv"
+    fo_input_file_base = f"fo_input_{RUN_NAME}_{obj_id}"
+    fo_output_file_base = f"fo_output_{RUN_NAME}_{obj_id}"
+
+    impact_date = impactor_orbit.coordinates.time.add_days(30)
```

Comment:
I don't think we should change this now, but when we have our new population we may have to change this. I think it may require a new qv.Table that has `Orbits` as a subcolumn and has an `impact_date = qv.Timestamp(...)` column. Or we have to always define our population orbits to be defined 30 days out from their impact date.